system_metrics_collector/test/system_metrics_collector/test_periodic_measurement_node.cpp

```diff
@@ -136,19 +145,24 @@ TEST_F(PeriodicMeasurementTestFixure, test_start_and_stop) {
 
   rclcpp::executors::SingleThreadedExecutor ex;
   ex.add_node(test_periodic_measurer);
-  ex.spin_until_future_complete(dummy_future, TEST_LENGTH);
+  ex.spin_until_future_complete(dummy_future, test_constants::TEST_LENGTH);
 
-  const moving_average_statistics::StatisticData data =
-    test_periodic_measurer->getStatisticsResults();
-  ASSERT_EQ(3, data.average);
-  ASSERT_EQ(1, data.min);
-  ASSERT_EQ(TEST_LENGTH.count() / TEST_PERIOD.count(), data.max);
-  ASSERT_FALSE(std::isnan(data.standard_deviation));
-  ASSERT_EQ(TEST_LENGTH.count() / TEST_PERIOD.count(), data.sample_count);
+  moving_average_statistics::StatisticData data = test_periodic_measurer->getStatisticsResults();
```

Comment:
ASSERT was used purposely as it's highly likely all the other tests will fail given the first failure. I'd prefer to see these changes reverted.

Reply:
Okay, I have removed this change from this PR.